docs/references/sdk/terminology.mdx

```diff
@@ -19,7 +19,27 @@ Across the complete surface area that users interact with Clerk’s services (e.
 | Instances | When creating an app, you’re provided with two instances: Development and Production. Learn more. |
 | Hotloading clerk-js | `@clerk/clerk-js` is the foundational JavaScript library for all frontend JS SDKs, e.g. used in `@clerk/clerk-react`. When you install `@clerk/clerk-react` through npm, you don’t install `@clerk/clerk-js`. Instead, once the React code is executed in the browser, the React SDK adds a `<script>` tag to load clerk-js from a CDN. Clerk internally calls this “hotloading”. |
 | UI components | All components displayed in the component reference and available through the Clerk class. |
-| `__client` cookie | A long-lived client JWT that is used to keep the session JWT fresh by interacting with the FAPI. |
-| `__session` cookie | A short-lived session JWT to validate request in your application or your API. |
 | Authorization header verification | This refers to the HTTP Authorization request header. Clerk uses this HTTP header instead of the `__client` cookie for secure communication with the FAPI in non-standard web browser environments. For example in React Native apps, Chrome extensions, and hybrid apps like Electron. The `__client` JWT will be stored in a secure storage provided by the platform and injected into the HTTP header of each FAPI request. |
 | Handshake | The client handshake is a mechanism that is used to resolve a request’s authentication state from “unknown” to definitively signed in or signed out. Clerk’s session management architecture relies on a short-lived session JWT to validate requests, along with a long-lived session that is used to keep the session JWT fresh by interacting with the FAPI. The long-lived session token is stored in an `HttpOnly` cookie associated with the FAPI domain. If a short-lived session JWT is expired on a request to an application’s backend, the SDK doesn’t know if the session has ended, or if a new short-lived JWT needs to be issued. When an SDK gets into this state, it triggers the handshake. |
+
+## Cookies
+
+Clerk is using cookies in order to keep user sessions alive. The most important cookies are the following:
+
+### Development instance cookies
+
+| Cookie Name | Description |
+| - | - |
+| `__clerk_db_jwt` | This is called `Dev Browser JWT` and helps Clerk sync the session between different domains, because in the dev environment the app is running on `localhost` and Clerk FAPI and Account Portal runs under the `accounts.dev` domain. |
```

Comment:
```suggestion
| `__clerk_db_jwt` | When using the Development instance, the **Development Browser JWT** helps Clerk sync the session between different domains (`localhost` and `https://name.accounts.dev`) |
```

We mentioned "Instances" above, so keeping this in sync here. Also reworded a bit to make it more concise

Reply:
```suggestion
| `__clerk_db_jwt` | The **Development Browser JWT**. Helps Clerk sync the session between the development instance domain, `localhost`, and the Clerk FAPI and Account Portal domain, `https://name.accounts.dev`. |
```